TCAPractice/Graph/View/ChartView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ChartView.swift
+//  TCAPractice
+//
+//  Created by Raymond Chen on 1/5/24.
+//
+
+import ComposableArchitecture
+import SwiftUI
+import Charts
+
+struct ChartViewFeature: Reducer {
```

Comment:
Let's move this reducer to TCAReducerCore

Reply:
Can you also add some unit tests for the reducer? Let me know if you need help coming up with some test cases.